src/back/spv/writer.rs

```diff
@@ -1158,6 +1158,28 @@ impl Writer {
                             ),
                             left_lookup_ty,
                         ),
+                        (Dimension::Vector, Dimension::Vector)
+                            if left_ty_inner.scalar_kind() == Some(crate::ScalarKind::Float) =>
+                        {
+                            (
```

Comment:
this is formatted inconsistently with the follow-up arm, can we remove  the `{` and `}` around the tuple?

Reply:
Yes I can, `cargo fmt` put those `{` and `}` around it. Let me check if I remove it what happens.